commands.py

```diff
@@ -329,13 +347,21 @@ def repl_open(self, cmd_list, environment, syntax=None):
         if syntax is None:
             syntax = self.settings.get('repl_syntax')
 
+        syntaxpath = "Packages/%s.tmLanguage"
+        if syntax == "python":
+            syntaxpath = syntaxpath % "Python/Python"
+        elif syntax == "plaintext":
+            syntaxpath = syntaxpath % "Text/Plain text"
```

Comment:
We should be using `.format()` syntax rather than `%` syntax for string formatting.